mmv1/products/cgc/terraform.yaml

```diff
@@ -416,6 +416,16 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           new_bucket: "new-bucket"
           new_object: "new-object"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "storage_make_data_public"
+        primary_resource_id: "default"
```

Comment:
I think this is missing `primary_resource_type` similar to above.

The test fails with:
=== CONT  TestAccCGCSnippet_storageMakeDataPublicExample
    provider_test.go:309: Resource specified by ResourceName couldn't be found: google_cgc_snippet.default

Reply:
Thanks @slevenick. Just added it